tests/integration/test_core.py

```diff
@@ -131,13 +154,12 @@ async def test_openstack_object_storage_plugin(
     """
     if swift_config is None:
         pytest.skip("no openstack configuration provided, skip openstack swift plugin setup")
-    assert ops_test.model
-    application = ops_test.model.applications[application_name]
+    application = model.applications[application_name]
     await application.set_config(
         {"wp_plugin_openstack-objectstorage_config": json.dumps(swift_config)}
     )
     # mypy has trouble to inferred types for variables that are initialized in subclasses.
-    await ops_test.model.wait_for_idle(status=ops.model.ActiveStatus.name)  # type: ignore
+    await model.wait_for_idle(status=ops.model.ActiveStatus.name)  # type: ignore
```

Comment:
It is tempting to define a constant somewhere like `ACTIVE_STATUS_NAME` so that we don't have to litter `type: ignore` everywhere. Thoughts? Perhaps worth a standard in the contributing guide?

Reply:
That does sound pretty neat! Thank you for the suggestion!